ddtrace/contrib/coverage/__init__.py

```diff
@@ -15,26 +15,21 @@
 
 Note: Coverage.py instrumentation is only enabled if `pytest` or `unittest` instrumentation is enabled.
 """
-from ddtrace.internal.logger import get_logger
-from ddtrace.internal.utils.importlib import require_modules
-
+# Required to allow users to import from  `ddtrace.contrib.internal.coverage.patch` directly
+import warnings as _w  # noqa:E402
 
-required_modules = ["coverage"]
-log = get_logger(__name__)
 
+with _w.catch_warnings():
+    _w.simplefilter("ignore", DeprecationWarning)
+    from . import patch as _  # noqa: F401, I001
 
-with require_modules(required_modules) as missing_modules:
-    if not missing_modules:
-        # Required to allow users to import from `ddtrace.contrib.internal.coverage.patch` directly
-        import warnings as _w
+# Expose public methods
+from ddtrace.contrib.internal.coverage.patch import get_version
+from ddtrace.contrib.internal.coverage.patch import patch
+from ddtrace.contrib.internal.coverage.patch import unpatch
+from ddtrace.internal.logger import get_logger
 
-        with _w.catch_warnings():
-            _w.simplefilter("ignore", DeprecationWarning)
-            from . import patch as _  # noqa: F401, I001
 
-        # Expose public methods
-        from ddtrace.contrib.internal.coverage.patch import get_version
-        from ddtrace.contrib.internal.coverage.patch import patch
-        from ddtrace.contrib.internal.coverage.patch import unpatch
+log = get_logger(__name__)
```

Comment:
Probably some copy-pasta. It seems to have been there since the beginning of this file's history, and was already unused back then. :stuck_out_tongue: We can probably remove it.